src/ConnectedMode/Migration/MSBuildFileCleaner.cs

```diff
@@ -18,33 +18,26 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
-using System;
 using System.ComponentModel.Composition;
 using System.Threading;
 using System.Threading.Tasks;
-using EnvDTE;
 using SonarLint.VisualStudio.Integration;
 
 namespace SonarLint.VisualStudio.ConnectedMode.Migration
 {
-    [Export(typeof(IProjectCleaner))]
+    [Export(typeof(IFileCleaner))]
```

Comment:
I dropped the `RuleSetCleaner` and `IAdditionalFilesCleaner` and added skeleton implementations of the two new interfaces instead.